### PR TITLE
making page paragraphs work better when used as auto-generated landing page descriptions

### PR DESCRIPTION
**Pull request by chrisdmills-okta**
## Description:
- **What's changed?** There are a few landing pages that don't work very well with the new system of auto-generating landing pages (with sub-page links and descriptions) that forms part of the new tree nav we are launching soon. This PR makes some small updates to those pages, to make them work with the system.
- **Is this PR related to a Monolith release?** No

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/reference/telephony-hook/index.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ excerpt: Customizes Okta's flows that send SMS or Voice messages
 
 # Telephony Inline Hook Reference
 
-This page provides reference documentation for:
+This page provides reference documentation for Telephony Inline Hooks.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/telephony-hook/index.md:9]: The original stem introduced the bulleted items right below. There is no connection now to what those bulleted items are telling you. The title is just above that stem sentence, was repeating the title intentional? I suggest we leave it as it was or try one of these two suggestions:

This page provides reference documentation for Telephony Inline Hooks. It provides:

OR

This page provides reference documentation for Telephony Inline Hooks. It provides sample JSON objects that are contained in the outbound request from Okta to your external service, and sample JSON objects that you can include in your response.

```suggestion
This page provides reference documentation for:
```

File: packages/@okta/vuepress-site/docs/reference/token-hook/index.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ excerpt: Customize tokens returned by the Okta API Access Management process flo
 
 # Token Inline Hook Reference
 
-This page provides reference documentation for:
+This page provides reference documentation for Token Inline Hooks.
```
Comment (susanharper-okta) [inline packages/@okta/vuepress-site/docs/reference/token-hook/index.md:8]: The original stem introduced the bulleted items right below. There is no connection now to what those bulleted items are telling you. The title is just above that stem sentence, was repeating the title intentional? I suggest we leave it as it was or try one of these two suggestions:

This page provides reference documentation for Token Inline Hooks. It provides:

OR

This page provides reference documentation for Token Inline Hooks. It provides sample JSON objects that are contained in the outbound request from Okta to your external service, and sample JSON objects that you can include in your response.

```suggestion
This page provides reference documentation for:
```

**Review by susanharper-okta — COMMENTED** [review]
handful of nits and some suggestions on one change in multiple documents.

**Comment by chrisdmills-okta** [comment]
Thanks for the comments @susanharper-okta ! I've rewritten those hooks ref paragraphs according to your idea, as it works well, but I've also aded an extra bit that means I cna delete the rather repetitious following paragraph in each case.

**Review by susanharper-okta — APPROVED** [review]
LGTM 🚀

**Pull request merged**